src/lib/types.ts

```diff
@@ -25,13 +24,22 @@ export enum ToolState {
   move
 }
 
-export interface Coordinate {
+export interface Rectangle {
+	width: number;
+	height: number;
+}
+
+export interface Coordinates {
```

Comment:
```suggestion
export interface Coordinate {
```
Denne refererer vel til et enkelt koordinat. ikke flere?

Reply:
For kontekst til andre der kigger: Det viser sig at x og y hver i sær hedder et koordinat, så et koordinatsæt er koordinater